lib/Value/Matrix.pm

```diff
@@ -6,128 +6,137 @@
 
 =head1 Value::Matrix class
 
-
 References:
 
-MathObject Matrix methods: L<http://webwork.maa.org/wiki/Matrix_(MathObject_Class)>
-MathObject Contexts: L<http://webwork.maa.org/wiki/Common_Contexts>
-CPAN RealMatrix docs: L<http://search.cpan.org/~leto/Math-MatrixReal-2.09/lib/Math/MatrixReal.pm>
+=over
 
-Allowing Matrices in Fractions:
-L<http://webwork.maa.org/moodle/mod/forum/discuss.php?d=2978>
+=item MathObject Matrix methods: L<http://webwork.maa.org/wiki/Matrix_(MathObject_Class)>
 
-     Context()->parens->set("[" => {formMatrix => 1});
+=item MathObject Contexts: L<http://webwork.maa.org/wiki/Common_Contexts>
 
-Files interacting with Matrices:
+=item CPAN RealMatrix docs: L<http://search.cpan.org/~leto/Math-MatrixReal-2.09/lib/Math/MatrixReal.pm>
 
-L<MatrixReal1>
+=back
 
-L<MatrixReduce.pl>
+For allowing Matrices in Fractions, see L<http://webwork.maa.org/moodle/mod/forum/discuss.php?d=2978>
 
-L<Matrix>
+    Context()->parens->set("[" => {formMatrix => 1});
 
-L<MatrixCheckers.pl> -- checking whether vectors form a basis
+=head2 Files interacting with Matrices:
 
-L<MatrixReduce.pl>  -- tools for  row reduction via elementary matrices
+=over
 
-L<MatrixUnits.pl>   -- Generates unimodular matrices with real entries
+=item L<MatrixReal1>
 
-L<PGmatrixmacros.pl>
+=item L<MatrixReduce.pl>
 
-L<PGmorematrixmacros.pl>
+=item L<Matrix>
 
-L<PGnumericalmacros.pl>
+=item L<MatrixCheckers.pl> -- checking whether vectors form a basis
 
-L<tableau.pl>
+=item L<MatrixReduce.pl>  -- tools for  row reduction via elementary matrices
 
-quickMatrixEntry.pl
+=item L<MatrixUnits.pl>   -- Generates unimodular matrices with real entries
 
-L<LinearProgramming.pl>
+=item L<PGmatrixmacros.pl>
 
-Contexts
+=item L<PGmorematrixmacros.pl>
 
-	Matrix -- allows students to enter [[3,4],[3,6]]
-	       -- formMatrix =>1 also allows this?
-	Complex-Matrix -- allows complex entries
+=item L<PGnumericalmacros.pl>
 
-Creation methods
+=item L<tableau.pl>
 
-		 $M1 = Matrix([1,2],[3,4]);
-   		 $M2 = Matrix([5,6],[7,8]);
-    	  $v = Vector(9,10);
-    	  $w = ColumnVector(9,10); # differs in how it is printed
+=item quickMatrixEntry.pl
 
-Commands added in Value::matrix
+=item L<LinearProgramming.pl>
+
+=back
+
+=head2 Contexts
 
-	Conversion
-		$matrix->values produces [[3,4,5],[1,3,4]] recursive array references of numbers (not MathObjects)
-		$matrix->wwMatrix   produces CPAN MatrixReal1 matrix, used for computation subroutines
+=over
+=item C<Matrix> -- allows students to enter [[3,4],[3,6]]
+				 -- formMatrix =>1 also allows this?
+=item C<Complex-Matrix> -- allows complex entries
 
-	Information
-		$matrix->dimension:  ARRAY
+=back
 
-	Access values
+=head2 Creation of Matrices
 
-		row : MathObjectMatrix
-		column : MathObjectMatrix
-		element : Real or Complex value
+Using the C<Matrix>, C<Vector> or C<ColumnVector> methods
+
+Examples:
+
+    $M1 = Matrix([1,2],[3,4]);
+    $M2 = Matrix([5,6],[7,8]);
+    $v = Vector(9,10);
```

Comment:
Do the Vector and ColumnVector examples really belong here? It seems misleading, since those are distinct MathObjects from a Matrix.